JustSaying/JustSayingBus.cs

```diff
@@ -167,22 +167,21 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
             {
                 activeRegion = Config.GetActiveRegion();
             }
-            _log.LogInformation($"Active region has been evaluated to {activeRegion}");
+            _log.LogInformation("Active region has been evaluated to '{Region}'.", activeRegion);
 
             if (!_publishersByRegionAndTopic.ContainsKey(activeRegion))
```

Comment:
Unrelated to logging perf improvement. Should use `TryGetValue()` here so that the call to the indexer on line 179 isn't needed.